temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -234,3 +235,89 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetNexusEndpointsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+
+    // Filter endpoints based on their target specification - optional, treated as an AND if specified. Empty fields within the spec are treated as '*'.
+    temporal.api.cloud.nexus.v1.EndpointTargetSpec spec = 3;
+
+    // Filter endpoints by their name - optional, treated as an AND if specified. Specifying this will result in zero or one results.
+    string name = 4;
+}
+
+message GetNexusEndpointsResponse {
+    // The list of endpoints in ascending id order.
+    repeated temporal.api.cloud.nexus.v1.Endpoint endpoints = 1;
+
+    // The next page's token.
+    string next_page_token = 2;
+}
+
+message GetNexusEndpointRequest {
+    // The id of the nexus endpoint to get.
+    string endpoint_id = 1;
```

Comment:
I think you can just call this `id`, that's what I did in the operator API.

Reply:
so I changed this to id, but this is now in-consistent with our other APIs related to user and namespace operations (where the request/path takes in user_id or namespace_id). am fine with having this as id, but will check with @anekkanti to be sure

Reply:
changed it back